src/lib/mixins/FlowUploader.js

```diff
@@ -0,0 +1,75 @@
+import Flow from '@flowjs/flow.js'; // todo: can we use this ? or need to upgrade php lib (voto5 legacy todo)
```

Comment:
I just copied from voto5, do we need to update todo descriptions here ?

Reply:
In our previous repo, we were using a globally included stale version of this library. We've now updated to use the npm version. We'll need to update the endpoint in our consumer repo to handle this newer library version if any changes are necessitated.

Reply:
**note**: remove this comment from here

Reply:
removed + add note in readme